### PR TITLE
Release 2.05.0.1.0 merge

### PR DESCRIPTION
**Pull request by pavitthrap**
Merged changes from master into the side branch. 

This branch was created to resolve merge conflicts between develop and master for release 2.05.0.1.0.

### CODE REVIEW

**Review by jcnelson — APPROVED** [review]
LGTM!

File: README.md [hunk 1]
```diff
@@ -463,7 +462,10 @@ is tagged.
 
 1. Once the final release candidate has rolled out successfully without issue on the
    above staging infrastructure, the release manager tags 2 additional `stacks-blockchain`
-   team members to review the `develop -> master` PR.
+   team members to review the `develop -> master` PR. If there is a merge conflict in this
```
Comment (gregorycoppola) [inline README.md:465]: is this new code?

**Review by kantai — APPROVED** [review]
LGTM -- thanks for merging these changes.

File: README.md (hunk 1 above)
Comment (kantai) [reply README.md:465]: No -- this is just a merge from `master` to `develop`, so all these changes are already in `master`. The code changes are limited to the benchmark fixes.

**Review by gregorycoppola — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (pavitthrap) [reply README.md:465]: I think having docs changes does not count as new code.

**Pull request merged**